testing/test_mark.py

```diff
@@ -563,6 +563,29 @@ def assert_markers(self, items, **expected):
                                 if isinstance(v, MarkInfo)])
             assert marker_names == set(expected_markers)
 
+    @pytest.mark.xfail(reason='callspec2.setmulti missuses keywords')
```

Comment:
nitpick: missuses -> misuses

(TIL ["missuses" is a colloquial form of "wives"](https://en.wiktionary.org/wiki/missus#English) too)